### ISSUE TITLE
Normalisation is performed after linear translation and non-linear operation

### ISSUE THREAD

**Issue by yinyee** [twistedcubic/HNHN#2]
For example, with respect to edge convolution, a linear translation `+ self.b_v` and a non-linear operation `F.relu()` occur at line 50 in hypergraph.py _before_ normalisation in lines 58 and 60.  In fact, even the pooling of messages from neighbours (i.e., matrix multiplication the with incidence matrix _A_) occurs afterwards at line 59.

Similarly, with respect to vertex convolution at line 62 in the same function.

This is different to the order of computation in **Algorithm 1** found in the paper, which states that normalisation is performed prior to linear translation and nonlinear operation.
